cluster-up/cluster/kind/common.sh

```diff
@@ -172,6 +157,13 @@ function setup_kind() {
     docker cp ${CLUSTER_NAME}-control-plane:/kind/bin/kubectl ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
     chmod u+x ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
 
+    if [ $KUBEVIRT_WITH_KIND_ETCD_IN_MEMORY == "true" ]; then
+        echo "Checking KIND cluster etcd data is mounted to RAM: $ETCD_DATA_DIR"
+        docker exec "$CLUSTER_NAME-control-plane" df -h $(dirname $ETCD_DATA_DIR)
+        docker exec "$CLUSTER_NAME-control-plane" du -h $ETCD_DATA_DIR
```

Comment:
This will check that it's mounted but not that it's mounted in memory, can you check that it's using `tmpfs` ?

Reply:
No problem